mtags/src/main/scala-3/scala/meta/internal/pc/completions/MatchCaseCompletions.scala

```diff
@@ -426,22 +472,30 @@ class MatchCaseExtractor(
   object MatchExtractor:
     def unapply(path: List[Tree]) =
       path match
+        case (ident @ Ident(name)) :: Block(stats, expr) :: (appl @ Apply(
```

Comment:
other cases have comment which describes what is being matched on them, but this one is unlucky :)